src/sparseml/pytorch/torchvision/train.py

```diff
@@ -759,9 +759,10 @@ def _create_model(
             model, arch_key = model
     elif arch_key in torchvision.models.__dict__:
         # fall back to torchvision
-        model = torchvision.models.__dict__[arch_key](
-            pretrained=pretrained, num_classes=num_classes
-        )
+        if pretrained is not None:
+            model = torchvision.models.__dict__[arch_key](pretrained=pretrained)
+        else:
+            model = torchvision.models.__dict__[arch_key](num_classes=num_classes)
```

Comment:
I think the following will support pretrained with a different number of classes:
```suggestion
        model = torchvision.models.__dict__[arch_key](num_classes=num_classes)
        if pretrained is not None:
            pretrained_model = torchvision.models.__dict__[arch_key](pretrained=pretrained)
            model.load_state_dict(pretrained_model.state_dict(), strict=False)
```